.github/workflows/layer_rename.yml

```diff
@@ -0,0 +1,161 @@
+# Rename Layer
+# ---
+# This workflow copies a specific layer version in an AWS account, renaming it in the process
+#
+# Using a matrix, we pull each architecture and python version of the layer and store them as artifacts
+# we upload them to each of the AWS accounts.
+#
+# A number of safety checks are performed to ensure safety.
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: Deployment environment
+        type: choice
+        options:
+          - beta
+          - prod
+        default: Gamma
+        required: true
```

Comment:
Do we need this?

Reply:
I see it on line 126. But we are renaming prod to prod, we don't rename prod to beta for example, we don't need it. But anyway, please let me know if you want to keep this to run the pipeline and I'll approve.